api/app/signals/apps/feedback/models.py

```diff
@@ -18,11 +18,23 @@ def generate_token():
     return uuid.UUID(hex=secrets.token_hex(16))
 
 
+class StandardAnswerTopic(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    text = models.TextField(max_length=1000)
+    order = models.IntegerField(default=0, null=True, blank=True)
```

Comment:
Can you add a help text so that it will appear in the admin explaining how the sorting works.